gridfs/__init__.py

```diff
@@ -442,13 +442,16 @@ def __init__(self, db, bucket_name="fs",
             (the default) db.write_concern is used.
           - `read_preference` (optional): The read preference to use. If
             ``None`` (the default) db.read_preference is used.
-          - `disable_md5` (optional): When True, MD5 checksums will not be
-            computed for uploaded files. Useful in environments where MD5
-            cannot be used for regulatory or other reasons. Defaults to False.
+
+        .. versionchanged:: 4.0
+           Removed the `disable_md5` parameter.
```

Comment:
Same here.

Reply:
Updated to link to the migration guide which describes the change in more detail.